packages/addons-website/package.json

```diff
@@ -23,15 +23,19 @@
   },
   "devDependencies": {
     "@babel/core": "^7.2.2",
+    "babel-loader": "7.1.1",
     "@babel/preset-env": "^7.2.0",
     "@babel/preset-react": "^7.0.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
```

Comment:
Do we actually use dynamic import (`import()` as a function) syntax?

Reply:
removed this 👍